helm/zeno/values.yaml

```diff
@@ -1,3 +1,108 @@
+# Global configuration
+global:
+  environment: default
```

Comment:
Where is this used?

Reply:
ah, I don't believe it is. Nice catch, will remove.